### DISCUSSION TITLE
No Support

### DISCUSSION THREAD

**Discussion by grandpa3d** [visualdesigncafe/nature-shaders#11] [Help]
Question: Is this asset actively supported or is it dead. Getting no response to a reported bug. If this asset is dead just let me know and I'll find another asset to use in it's place.

**Comment by frankcarey** [comment]
Just to ping @maxplooi again here. I added more details to the github issue about the 2022.3 HDRP we are experiencing on the ticket @grandpa3d started. I'm hoping someone can confirm that HDRP 2022.3 is working for them and what versions if its working for anyone? I just purchased this pack to replace the old 2021 version I already had.  

https://github.com/visualdesigncafe/nature-shaders/issues/10
